pkg/virt-operator/kubevirt_test.go

```diff
@@ -1495,7 +1495,7 @@ var _ = Describe("KubeVirt Operator", func() {
 
 		}, 15)
 
-		It("should add resources on create", func() {
+		It("[test_id:2806]should add resources on create", func() {
```

Comment:
we don't need to add test attributes to unit tests, we can use it only inside kubevirt/tests/ folder

I think it's better to add steps from 2806 to other functional test

Reply:
OK, added.

Reply:
Also I removed the ref_id and test_id from unit test.

Reply:
Thank you!